examples/12-fluids/00-fluids_model.py

```diff
@@ -0,0 +1,185 @@
+"""
+.. _ref_fluids_model:
+
+Explore Fluids models
+------------------------------------------------------
+
+This example shows how to explore Ansys Fluent and Ansys CFX models employing
+the ``MeshInfo`` and ``ResultInfo``.
+"""
+
+###############################################################################
+# Exploring an Ansys Fluent model
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# The first part of the example demonstrates how you can explore an Ansys Fluent
+# model. Import the result file and create a model
+
+import ansys.dpf.core as dpf
+from ansys.dpf.core import examples
+
+path = examples.download_fluent_axial_comp()["flprj"]
+ds = dpf.DataSources(path)
+model = dpf.Model(data_sources=ds)
+
+
+###############################################################################
+# Exploring the mesh
+# ------------------
+# Explore the mesh through the ``MeshInfo``. The ``MeshInfo`` provides metadata
+# information about the mesh. For fluid models, it is useful to know the cell and
+# face zones, as well as the topological relationships between them. First get all
+# the available information in the ``MeshInfo``
+
+minfo = model.metadata.mesh_info
+minfo
+
+###############################################################################
+# Then, get the bodies and their names in the model with the "body_names" ``StringField``,
+# which provides a relationship between body IDs and names. In this model there are two
+# bodies.
+
+minfo.get_property("body_names")
+
+###############################################################################
+# Each body is comprised of a set of cell zones. You can investigate the hierarchical
+# relationship between bodies and cell zones through the "body_cell_topology"
+# ``PropertyField``, which provides a relationship between the body IDs and the cell zone
+# IDs. In this case, each body is only comprised of one cell zone.
+
+minfo.get_property("body_cell_topology")
```

Comment:
hey @rafacanton don't we want to expose realy APIs (with no strings) in the mesh info